m2m-transfers/m2m-transfers-json-contract.md

```diff
@@ -0,0 +1,238 @@
+				"transferName": "REGULAR CHECKING",
+				"available": "500.00"
+			},
+			{
+				"transferSLId": "0000800005S0015",
+				"transferName": "ULTIMATE CHECKING (10)",
+				"available": "0.00"
+			}
+		],
+
+		"scheduledTransfers": [{
+				"transferLoc": "215",
+				"sourceAccount": "1234567890S0020",
+				"accountName": "CHECKING",
+				"transferAmt": "120.00",
+				"recipientName": "Emily Fitch",
+				"recipientMemberId": "9876543210",
+				"recipientAccount": "9876543210L0001", // could be specific account, or "savings" or "checking"?
+				"recipientNickname": "Emmy",
+				"startDate": "07/31/21",
+				"endDate": "12/31/2021", // unused
+				"transferFrequency": "weekly", // daily, weekly, monthly, yearly, semiMonthly, biweekly, quarterly
+				"day1": "", // semi-monthly - first day
+				"day2": ""// semi-monthly - second day
+				
+			},
+			{
+				"transferLoc": "395",
+				"sourceAccount": "1234567890S0001",
+				"accountName": "SAVINGS",
+				"transferAmt": "120.00",
+				"recipientName": "Howard Cleo",
+				"recipientMemberId": "9876543210",
+				"recipientShareId": "9876543210L0001",
+				"recipientNickname": "Howie",
+				"nextTransferDate": "12/15/2020",
+				"startDate":"07/03/21",
+				"endDate":"12/31/2025", // unused
+				"transferFrequency": "semi-monthly",
+				"day1": "3",
+				"day2": "31"
+			}
+		],
+		"savedRedipients": [{
+				"recipientLoc": "5443231543",
+				"recipientName": "Binsy Flomor",
+				"recipientSLId": "9876543210L0001",// full account id or "checking" or "savings"?
+				"recipientNickName": "Zeke's Future"
+			},
+			{
+				"recipientLoc": "5431543",
+				"recipientName": "Kylie Crenshaw",
+				"recipientSLId": "9876543210L0001",
+				"recipientNickName": "Sally Martin"
+			}
+		]
+	}
+}
+```
+### PowerOn Error Responses (PRELOADDATA)
+#### PowerOn response - Configuration File read error:
+```json
+{
+  "errorCode": "501",
+  "loggingErrorMessage": "Error [opening/reading] from config file: [error msg]"
+}
+```
+#### PowerOn response - Configuration File read error:
+```json
+{
+  "errorCode": "501",
+  "loggingErrorMessage": "Error [opening/reading] from config file: [error msg]"
+}
+```
+### PowerOn response - Invalid Account (by Account Warning):
+
+```json
+{
+  "errorCode": "502",
+  "loggingErrorMessage": "Invalid Account - account warning xxx"
+}
+```
+### PowerOn response - No Eligible Share(s) found:
+
+```json
+{
+  "errorCode": "503",
+  "loggingErrorMessage": "No eligible shares found"
+}
+```
+## CREATE TRANSFER REQUEST STATE
+### Client Request (CREATETRAN)
+
+```json
+{
+  "rgState": "CREATETRAN",
+  "powerOnFileName": "BANNO.M2MTRANSFERS.V1.POW",
+  "userChrList": [
+    { "id": 1, "value": "395" }, // recipientLoc if available, empty if not
+    { "id": 2, "value": "1234567890S0001|9876543210L0001|weekly|12/31/2021|1|15" },  // [sourceAccount][destinationAccount][frequency]|[startDate]|[day1]|[day2]
+    { "id": 3, "value": "HUB|nickname" }, // [first 3][nickname]
```

Comment:
@tkainz do you need the first 3 letters of the members name? - and if not - do you know why they ask for the first 3 letters?

Reply:
The first 3 letters of the member's last name (or first 3 letters of the business name) are to validate that the transfer will be going to the correct account.